packages/web/src/scss/components/Icon/README.md

```diff
@@ -0,0 +1,15 @@
+# Icon
+
+Use Icon component to display an icon from our icon set. This icon set is
+exported from Supernova into separate `spirit-icons` package. So make sure you
+install it first and then copy the icons into your project.
+
+You can use sprite with all the icons.
```

Comment:
I'd either rephrase or remove this. Now it sounds like an option, but sprites are currently the _only_ way I can use icons.

Reply:
Removed